(18) Council data download page

CristinaRO opened the pull request:
## Changes in this PR
- Add links to data downloads for all the available local authorities
- Move the download link for the votes.tsv file into the new Data page
- Link the Data page from the FAQ

## Screenshots of UI changes

### After

#### Data page
<img width="1408" alt="Screenshot 2022-04-22 at 14 26 40" src="https://user-images.githubusercontent.com/579522/164723522-16bd34b9-2f6b-48dc-9e64-21836649274d.png">

#### FAQ page
<img width="908" alt="Screenshot 2022-04-22 at 14 27 12" src="https://user-images.githubusercontent.com/579522/164723597-bea2658e-8387-4ac2-b5ee-90f3dce2d6f4.png">

CristinaRO committed:
Add structure for a LocalAuthority model

CristinaRO committed:
Add seed for local authorities

CristinaRO committed:
Move the link to votes.tsv into the Data page

The FAQ links to the Data page as well.

CristinaRO commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Council data download page](https://trello.com/c/M3jBgjEu/18-council-data-download-page)

pezholio commented on db/schema.rb:
```diff
@@ -10,10 +10,16 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "local_authorities", force: :cascade do |t|
```
Given this data is pretty much static - I wonder if it's better to read from a config file?

CristinaRO replied:
The thinking was that it might not be static, and being database backed would give us flexibility in the future.

pezholio replied:
👍

CristinaRO committed:
Data page links to all the local authorities’ data

Generate downloads for local authorities from the hostname and the name
of the local authority. The naming convention was established by the
Data Science Lab, who are managing the generation and the uploading of
the data files.

pezholio reviewed (approved):
LGTM 👍
